### DISCUSSION TITLE
Use of Livewire in larger projects

### DISCUSSION THREAD

**Discussion by Rayiumir** [livewire/livewire#9083] [1. Help] [answered]
Hi...

Is Livewire used in larger projects? 
They say that it is more for the admin panel, it is not suitable for user pages.
I mean bigger projects with several million users.

**Comment by sueboy19** [comment, marked as answer]
I believe a website with several million users is not a problem, because Laravel and the controllers are still active and functional, not outdated.

From my experience, Livewire can sometimes be difficult to manage, so it's often better to move things to the controller or API, especially when integrating third-party payment systems.

For large-scale websites with millions of users, every page typically requires multiple components. However, some pages might only need one or two components, depending on the complexity. And let's not forget Alpine.js!

You can even render a Livewire page with minimal content and rely solely on Alpine.js to call the API, similar to how you would with Vue.js. Why not? You can leverage CDN caching for long-term performance, with manual cache clearing as needed.

As Livewire continues to evolve, don't limit your imagination — explore its full potential!

**Reply by Rayiumir** [reply]
Thank you for your answer.
It was perfect.
